Header header
float64 timestamp
StaticObstaclePoint[] static_point
DynamicObstacle[] dynamic_obstacle
